content/en/docs/reference/kubectl/jsonpath.md

```diff
@@ -98,4 +98,14 @@ kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{\"\t\"}{.status.
 ```
 {{< /note >}}
 
+{{< note >}}
+
+JSONPath RegEx are currently not supported. You must use jq or similar.
+Example currently not supported
+
+```shell
+kubectl get secrets -o jsonpath='{.items[?(@.metadata.name=~/^test$/)].metadata.name}'
```

Comment:
```suggestion
# kubectl does not suport regular expressions
# The following command does not work
kubectl get secrets -o jsonpath='{.items[?(@.metadata.name=~/^test$/)].metadata.name}'

# The following command achieves the same result
kubectl get secrets -o json | jq '[needs actual example code writing before merge]'
```

Reply:
Yes here it is 
`kubectl get secrets -o json  | jq '.items[].metadata.name' | grep '^test-.*'`

also we may want to add the error message

error: error parsing jsonpath {.items[?(@.metadata.name=~/^test$/)].metadata.name}, unrecognized character in action: U+007E '~'